### ISSUE TITLE
flex-2.6.4 build fails with segfault on Ubuntu 18.04

### ISSUE THREAD

**Issue by DeeprajPandey** [westes/flex#425]
Facing the following issue when I run make in the release copy.
`stage1scan.c` fails with a segmentation fault.
Can someone point me in the right direction here? Thanks!

```
Making all in src
make[1]: Entering directory '/home/deeprajpandey/flex-2.6.4/src'
make  all-am
make[2]: Entering directory '/home/deeprajpandey/flex-2.6.4/src'
./stage1flex   -o stage1scan.c ./scan.l
Makefile:1696: recipe for target 'stage1scan.c' failed
make[2]: *** [stage1scan.c] Segmentation fault (core dumped)
make[2]: Leaving directory '/home/deeprajpandey/flex-2.6.4/src'
Makefile:546: recipe for target 'all' failed
make[1]: *** [all] Error 2
make[1]: Leaving directory '/home/deeprajpandey/flex-2.6.4/src'
Makefile:534: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
```

**Comment by Explorer09** [comment]
Thank you, but this is a known issue of flex-2.6.4.
Duplicate of issue #241 (already fixed in master) and PR #262.
**Workaround:** `./configure CFLAGS='-g -O2 -D_GNU_SOURCE'`

**Comment by DeeprajPandey** [comment]
Thank you!
